tests/unit/client_regression.py

```diff
@@ -0,0 +1,56 @@
+"""Unit tests for the client library.
+
+TODO: we have plans to move ./client.py to integration/, since those
+are really integration tests. Once that's done we should move this to
+./client.py; it's here now to avoid name collisions/conflicts.
+"""
+
+import flask
+import pytest
+from schema import Schema
+
+from hil import config, rest
+from hil.client.base import FailedAPICallException
+from hil.client.client import Client
+from hil.test_common import HybridHTTPClient, fail_on_log_warnings, \
+    fresh_database, server_init, config_testsuite
+
+fail_on_log_warnings = pytest.fixture(fail_on_log_warnings)
+fresh_database = pytest.fixture(fresh_database)
```

Comment:
why is this fixture required?

Reply:
It isn't, I just add the "standard" fixtures without really thinking about it -- I've removed it.